metrics-operator/controllers/metrics/controller.go

```diff
@@ -142,3 +170,111 @@ func (r *KeptnMetricReconciler) fetchProvider(ctx context.Context, namespacedMet
 	}
 	return provider, nil
 }
+
+func aggregateValues(stringSlice []string, aggFunc string) (string, error) {
+	floatSlice, err := stringSliceToFloatSlice(stringSlice)
+	if err != nil {
+		return "", err
+	}
+	var aggValue string
```

Comment:
maybe we can use a float here and use `fmt.Sprintf` in the return
```suggestion
	var aggValue float64
```